.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       # Restore go module cache if there is one
       - restore_cache:
           keys:
-            - consul-ecs-acceptance-modcache-v1-{{ checksum "test/acceptance/go.mod" }}
+            - consul-ecs-acceptance-modcache-v2-{{ checksum "test/acceptance/go.mod" }}
```

Comment:
We had change the consul-ecs-test image to use the newer CircleCI image (`cimg/go`) which changes the go path, hence these other changes.

Reply:
TODO: I missed updating a spot below in this file where the modcache is restore (will update that after the current acceptance test run finishes)